nautobot/tenancy/views.py

```diff
@@ -82,22 +82,23 @@ class TenantView(generic.ObjectView):
 
     def get_extra_context(self, request, instance):
         stats = {
```

Comment:
Wonder if we can replace these with a for loop.

Reply:
Not sure I see a straightforward way to do that.